openmetadata-ui/src/main/resources/ui/src/components/TableProfiler/Component/ProfilerSettingsModal.tsx

```diff
@@ -546,66 +584,26 @@ const ProfilerSettingsModal: React.FC<ProfilerSettingsModalProps> = ({
                 </>
               )}
             </List>
-            <Form.Item className="m-b-xs">
-              <Space size={12}>
-                <p>{t('label.enable-partition')}</p>
-                <Switch
-                  checked={state?.enablePartition}
-                  data-testid="enable-partition-switch"
-                  disabled={isPartitionDisabled}
-                  onChange={handleEnablePartition}
-                />
-              </Space>
-            </Form.Item>
             <Row gutter={[16, 16]}>
-              <Col span={12}>
-                <Form.Item
-                  className="m-b-0"
-                  label={
-                    <span className="text-xs">
-                      {t('label.column-entity', {
-                        entity: t('label.name'),
-                      })}
-                    </span>
-                  }
-                  labelCol={{
-                    style: {
-                      paddingBottom: 8,
-                    },
-                  }}
-                  name="partitionColumnName"
-                  rules={[
-                    {
-                      required: state?.enablePartition,
-                      message: t('message.field-text-is-required', {
-                        fieldText: t('label.column-entity', {
-                          entity: t('label.name'),
-                        }),
-                      }),
-                    },
-                  ]}>
-                  <Select
-                    allowClear
-                    className="w-full"
-                    data-testid="column-name"
-                    disabled={!state?.enablePartition}
-                    options={partitionColumnOptions}
-                    placeholder={t('message.select-column-name')}
-                    size="middle"
-                  />
-                </Form.Item>
+              <Col span={24}>
+                <Space align="center" size={12}>
+                  <p>{t('label.enable-partition')}</p>
+                  <Form.Item className="m-b-0" name="enablePartitioning">
```

Comment:
```suggestion
                  <Form.Item label={t('label.enable-partition')} className="m-b-0" name="enablePartitioning">
```

Reply:
Our form has a layout type Vertical and we need a horizontal layout for the switch.